0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -311,10 +178,11 @@ Content-Type: application/json
   "subject": "acct:bob@red.ilpdemo.org",
   "links": [
     {
-      "rel": "https://interledger.org/rel/receiver",
+      "rel": "https://interledger.org/rel/spsp",
```

Comment:
The current SPSP client in `ilp` looks for `https://interledger.org/rel/spsp/v1`. Should we change it there, or here?